lib/usb_connection.js

```diff
@@ -110,6 +110,7 @@ USB.Connection.prototype.open = function() {
     // Try to open connection
     try {
       self.device.open();
+      self.closed = false;
```

Comment:
We should explicitly mark a Tessel as not closed when it's opened so you can open and close multiple times from a single API consumer.